linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_privileged_commands_init/rule.yml

```
@@ -0,0 +1,32 @@
+documentation_complete: true
+
+title: 'Ensure auditd Collects Information on the Use of Privileged Commands - init'
+
+description: |-
+    At a minimum, the audit system should collect the execution of
+    privileged commands for all users and root. If the <tt>auditd</tt> daemon is
+    configured to use the <tt>augenrules</tt> program to read audit rules during
+    daemon startup (the default), add a line of the following form to a file with
+    suffix <tt>.rules</tt> in the directory <tt>/etc/audit/rules.d</tt>:
+    <pre>-a always,exit -F path=/usr/sbin/init {{{ perm_x }}}-F auid&gt;={{{ auid }}} -F auid!=unset -F key=privileged</pre>
+    If the <tt>auditd</tt> daemon is configured to use the <tt>auditctl</tt>
+    utility to read audit rules during daemon startup, add a line of the following
+    form to <tt>/etc/audit/audit.rules</tt>:
+    <pre>-a always,exit -F path=/usr/sbin/init {{{ perm_x }}}-F auid&gt;={{{ auid }}} -F auid!=unset -F key=privileged</pre>
+
+rationale:
+    Misuse of the init command may cause availability issues for the system.
+
+severity: medium
+
+references:
+    disa: CCI-000172
+    nist: AU-12 c
```

```suggestion
    nist: AU-12(c)
```

I don't if the blank space would cause any major issue, but maybe we should just encapsulate the `c` between parenthesis the same we do in:

https://github.com/ComplianceAsCode/content/blob/ae40a660b00634472d60d33a8b32f354dae21c9b/linux_os/guide/system/auditing/auditd_configure_rules/audit_execution_selinux_commands/audit_rules_execution_chcon/rule.yml#L54